solutionbox/structured_data/mltoolbox/_structured_data/trainer/task.py

```diff
@@ -152,9 +152,8 @@ def parse_arguments(argv):
                       type=str,
                       required=True,
                       help=('Output folder of preprocessing. Should contain the'
-                            ' files input_features.json, schema.json, and the'
-                            ' optional files numerical_analysis.json and'
-                            ' vocab_str1.csv. Path must be on GCS if running'
+                            ' schema file, and numerical stats and vocab files.'
+                            ' Path must be on GCS if running'
                             ' cloud training.'))
   parser.add_argument('--transforms-file',
```

Comment:
yea, everything in the trainer calls it transform. Users don't call the task.py file, but they will see --transforms-file if they look at the jobs page in the web UI. I figure when I stop using tf.learn, I'll have to rewrite task.py anyway, so I'll deal with it then.